src/content/lodash/lang/isEmpty/vanilla.js

```diff
@@ -0,0 +1,30 @@
+const isEmpty = (v) =>
```

Comment:
I believe this implementation is slightly more complex than it could be: not sure if you need to specify all the types and check against all of them when you are actually only after `length`, `size` and the length of Object.keys. I would rather focus on those as most of the use cases you cover with the other checks would be caught anyway (for instance a function doesn't really have a length or a size property, nor does a number).

The main problem I see in listing all the cases as this implementation does is that it's easy to miss something out (for example the boolean on line 4 is wrong, as `isEmpty(false)` should return `true`). I would also love if you could add the test for a Symbol since you added it in the implementation, but removing that line would still make the test pass

Reply:
JFYI: 

> a function doesn't really have a length or a size property

functions actually do have length property: it shows amount of the arguments (ignoring variadic args)

Reply:
> functions actually do have length property: it shows amount of the arguments (ignoring variadic args)

you are absolutely right 🤦